Add the PBC hopper results to the hopper example (#675)
lethe-cfd/lethe: doc/source/examples/dem/rectangular-hopper/rectangular-hopper.rst

This example simulates the filling and discharging of particles in a rectangular hopper.
We set up this simulation based on the simulation of Anand et al. `[1] <https://doi.org/10.1016/j.ces.2008.08.015>`_. It is recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_ for more detailed information on the concepts and physical meanings of the parameters in Lethe-DEM.

> Main example do not use periodic boundary conditions in depth of the hopper as the article, but the extensional case does.

Reviewer: ```suggestion
The main example does not use periodic boundary conditions in the depth of the hopper like article, but an extended case, presented at the end, does.
```

.. code-block:: text

    subsection insertion info
        set insertion method                               = non_uniform
        set inserted number of particles at each time step = 2910
        set insertion frequency                            = 25000
        set insertion box minimum x                        = -0.1030
        set insertion box minimum y                        Rectangular hopper at the end of the simulation.   = 0.10644
        set insertion box minimum z                        = .00224
        set insertion box maximum x                        = 0.1030
        set insertion box maximum y                        = 0.16020
        set insertion box maximum z                        = 0.03136
        set insertion distance threshold                   = 1.5
        set insertion random number range                  = 0.1
        set insertion random number seed                   = 20
    end

Results comparison
~~~~~~~~~~~~~~~~~~
Here is the comparison of the results from the original simulation with Lethe DEM, the simulation with periodic boundary conditions with Lethe and the results from Anand et al. paper.

> The simulated mass discharging rate is 84.94 g/s from the original simulation and 88.77 g/s with PBC.

Author: ```suggestion
The simulated mass discharging rate is 84.94 g/s from the original simulation and 88.77 g/s with PBC. Also, the run time of the simulation goes from about 3 hours and 15 minutes to 30 minutes on 8 cores.
```

.. figure:: images/figure_hopper_comparison.png
    :width: 600
    :alt: Comparison
    :align: center

>     Comparison of mass discharge results from the 2 simulations and the paper.

Reviewer: ```suggestion
    Comparison of mass discharge results from the 2 simulations and the journal article.
```

> 

Reviewer: Can you maybe add a comparison of the run time? I presume the periodic case has a much lower run time no?

Author: Yes from about 3h to 30 min
